pkg/util/tracing/collector/collector.go

```diff
@@ -53,59 +50,125 @@ func New(
 	}
 }
 
-// GetSpanRecordingsFromCluster returns the inflight span recordings from all
-// nodes in the cluster.
-// This method does not distinguish between requests for local and remote
-// inflight spans, and relies on gRPC short circuiting local requests.
-func (t *TraceCollector) GetSpanRecordingsFromCluster(
-	ctx context.Context, traceID uint64,
-) ([]tracingpb.RecordedSpan, error) {
-	var res []tracingpb.RecordedSpan
-	ns, err := nodesFromNodeLiveness(ctx, t.nodeliveness)
-	if err != nil {
-		return res, err
+// TraceCollectorIterator can be used to return RecordedSpans from all live
```

Comment:
s/TraceCollectorIterator/Iterator, it's fine to drop the package prefix (tracing/collector).

Reply:
done.